src/crimson/os/seastore/segment_cleaner.h

```diff
@@ -690,6 +701,8 @@ class SegmentCleaner : public SegmentProvider {
     extent_len_t len) {
     if (!init_complete)
       return;
+    if (addr.get_addr_type() != addr_types_t::SEGMENT)
+      return;
```

Comment:
This and mark_space_used make sense since SegmentCleaner won't be tracking space usage for RBM devices.